nuxeo-retention-web/elements/nuxeo-retention-analytics.js

```diff
@@ -0,0 +1,351 @@
+/**
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import moment from '@nuxeo/moment';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { I18nBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-i18n-behavior.js';
+import { FormatBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-format-behavior.js';
+import '@nuxeo/nuxeo-dataviz-elements/nuxeo-repository-data.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-card.js';
+import '@nuxeo/chart-elements/chart-bar.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-date-picker.js';
+
+/**
+`nuxeo-retention-analytics`
+@group Nuxeo UI
+@element nuxeo-retention-analytics
+*/
+class RetentionAnalytics extends mixinBehaviors([I18nBehavior, FormatBehavior], Nuxeo.Element) {
+  static get template() {
+    return html`
+      <style include="nuxeo-styles">
+        .infoCard {
+          @apply --layout-horizontal;
+          @apply --layout-justified;
+          @apply --layout-baseline;
+        }
+
+        .rangeSelector {
+          @apply --layout-horizontal;
+          @apply --layout-baseline;
+        }
+
+        .dateLabel {
+          margin-inline-end: 8px;
+        }
+
+        .flex-layout {
+          @apply --layout-wrap;
+          @apply --layout-horizontal;
+          margin: 0 -1em;
+          padding: 0 8px;
+          box-sizing: content-box;
```

Comment:
`content-box` is the default value of the property `box-sizing`. Why do we need to declare it here?

Reply:
good point! removed.